src/Sylius/Bundle/ThemeBundle/Asset/Package/PathPackage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Asset\Package;
+
+use Sylius\Bundle\ThemeBundle\Asset\PathResolverInterface;
+use Sylius\Bundle\ThemeBundle\Context\ThemeContextInterface;
+use Symfony\Component\Asset\Context\ContextInterface;
+use Symfony\Component\Asset\PathPackage as BasePathPackage;
+use Symfony\Component\Asset\VersionStrategy\VersionStrategyInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class PathPackage extends BasePathPackage
+{
+    /**
+     * @var ThemeContextInterface
+     */
+    protected $themeContext;
+
+    /**
+     * @var PathResolverInterface
+     */
+    protected $pathResolver;
+
+    public function __construct(
+        $basePath,
+        VersionStrategyInterface $versionStrategy,
+        ContextInterface $context = null,
+        ThemeContextInterface $themeContext,
+        PathResolverInterface $pathResolver
+    ) {
+        parent::__construct($basePath, $versionStrategy, $context);
+
+        $this->themeContext = $themeContext;
+        $this->pathResolver = $pathResolver;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getUrl($path)
+    {
+        if ($this->isAbsoluteUrl($path)) {
+            return $path;
+        }
+
+        $assetPath = $path;
+        if (null !== $this->themeContext) {
```

Comment:
We do not allow for null in the constructor, so there is no way the variable will be null.